data/summary/types/cgm.go

```diff
@@ -150,6 +159,12 @@ type CGMPeriod struct {
 	HasTimeInAnyHighRecords   bool `json:"hasTimeInAnyHighRecords" bson:"hasTimeInAnyHighRecords"`
 	TimeInAnyHighRecords      *int `json:"timeInAnyHighRecords" bson:"timeInAnyHighRecords"`
 	TimeInAnyHighRecordsDelta *int `json:"timeInAnyHighRecordsDelta" bson:"timeInAnyHighRecordsDelta"`
+
+	SD float64 `json:"SD" bson:"SD"`
```

Comment:
Consider not using an acronyms here

Reply:
Expanded variable names to be full names `StandardDeviation` and `CoefficientOfVariation`